trk/src/Navigation/AppNavigator.js

```diff
@@ -14,10 +16,35 @@ import SetDetail from '../Screens/SetDetail';
 import Settings from '../Components/Settings';
 import FeedbackForm from '../Screens/FeedbackForm';
 import GlossaryDefinition from '../Screens/GlossaryDefinition';
+import GymDaily from '../Screens/GymDaily';
 
 
 const Stack = createStackNavigator();
 const Tab = createBottomTabNavigator();
+const GymTopTab = createMaterialTopTabNavigator();
+
+function GymTabs() {
+  return (
+    <GymTopTab.Navigator
+      screenOptions={{
+        activeTintColor: '#3498db',
+        inactiveTintColor: 'gray',
+        swipeEnabled: true,
+      }}
+    >
+      <GymTopTab.Screen 
+        name="DailySummary"
+        component={GymDaily} 
+        options={{ title: 'Daily Summary' }}
+      />
+      <GymTopTab.Screen 
+        name="CompRanking"
+        component={CompRanking} 
+        options={{ title: 'Competition' }}
+      />
+    </GymTopTab.Navigator>
+  );
+}
 
```

Comment:
created a top tab navigator that swipes from left to right between the new screen 'Daily Summary' and the currently existing comp ranking